server/updateEmail.py

```diff
@@ -0,0 +1,33 @@
+import mysql.connector
+from mysql.connector import errorcode
+from connect import connectToDB
+
+
+def invalidEmail(email, cursor): 
+    checkEmailQuery = 'SELECT * FROM Users WHERE email = "'+ email +'"'
+    cursor.execute(checkEmailQuery)
+    if(cursor.rowcount > 0):
+        return True
+    return False
+
+def updateEmail(userID, newEmail):
+    try:
+        connection = connectToDB()
+        if(connection != False):
+            cursor = connection.cursor(buffered=True)
+
+            if(invalidEmail(newEmail, cursor)):
+                return {"response": "Email address is already in use"}
```

Comment:
If the user inputted the same email address as the new email address, should we throw an error suggesting the inputted email is the same instead of "Email address is already in use"?

Reply:
extra method added to check that in backend